module/userProjects/helper.js

```diff
@@ -2734,52 +2722,88 @@ module.exports = class UserProjectsHelper {
 	 * @returns {JSON} certificate data updation details.
 	 */
 
-	static certificateCallback(transactionId, osid) {
+	static certificateCallback(requestObject, projectId, userId) {
```

Comment:
@Prajwal17Tunerlabs first check the project is existing or not and get id and user id from it. later you can use same project id to update the data.

Reply:
check is added and userId , projectId is fetched from project.